### ISSUE TITLE
Don't send full notification text 

### ISSUE THREAD

**Issue by elad661** [holylobster/nuntius-android#30]
Right now nuntius sends the whole notification text from my phone to my computer. Often, when it comes to gmail notifications, this means the full text of an incoming email message.

It would be great if nuntius would cut the notification at a predefined length (an entire email in a notification does not make sense).

Also, email text can be sensitive, so if nuntius could allow me to select a list of apps of which it will hide the notification content (and only show the notification title) it'd be great.

**Comment by Kurtpenter** [comment]
You're definitely right. We are working on app blacklisting to completely silence notification for an app but we could introduce per app settings like the one you propose. 

Thanks for the report!

**Comment by elad661** [comment]
Even if I don't blacklist an app, it would make sense to cut the notification text at a certain length - Android itself doesn't display the entire text, it cuts it after few lines, so nuntius should do it too. 

Right now when I get an email the notification from nuntius covers my entire desktop screen :)

**Comment by Kurtpenter** [comment]
Yes, we should trim the content (or use the smaller text Android shows), and that's clear. But I'm intrigued by the privacy option you were proposing. You may have someone with you at your screen and still want to be notified without disclosing the contents.

**Comment by nacho** [comment]
@elad661 see though that it is a bug in the shell that is going to be fixed
